### ISSUE TITLE
Configuration | Guide | PM2 Plus Documentation

### ISSUE THREAD

**Issue by rmonnier9** [keymetrics/doc-pm2#208]
https://pm2.io/doc/en/plus/guide/configuration/ 

 PM2 - Guide, References, Tutorials and Best Practices about PM2 and Node.js.

**Comment by daton89** [comment]
So from now on instead of using pmx we should use @pm2/io?

**Comment by robin-monnier** [comment]
Exactly, `@pm2/io` will now replace the old `pmx`. We have improved and entirely recoded the module in Typescript.

**Comment by daton89** [comment]
> Exactly, @pm2/io will now replace the old pmx. We have improved and entirely recoded the module in Typescript.

So i cannot see the stats of my processes because i need to move to `@pm2/Io`?

![Image of keymetrics](https://preview.ibb.co/mAcY3T/Screen_Shot_2018_06_08_at_11_43_29.png)

**Comment by vmarchaud** [comment]
No, the old pmx should work without problem, thats a bug i think. You do you mean by `stats of my processes exactly`  : custom metrics or cpu/memory/latency etc ?

**Comment by daton89** [comment]
latency, rpm, errors. 
i'm using express with the pmx middleware for express... 
it worked well but from a while it doesn't work anymore... 
i'm a premium user and without these functionalities is like the free plan... 
i wrote the support but no one reply me..
